chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/load/JLoadProfileUserGroups.java

```diff
@@ -0,0 +1,48 @@
+package com.griddynamics.jagger.user.test.configurations.load;
+
+import java.util.List;
+import java.util.Objects;
+
+import static java.util.Collections.singletonList;
+
+public class JLoadProfileUserGroups {
+
+
+    private final List<JLoadProfileUsers> userGroups;
+
+    private int delayBetweenInvocationsInSeconds;
+
+    private int tickInterval;
+
+    public JLoadProfileUserGroups(JLoadProfileUsers userGroup) {
```

Comment:
can we use varags like in JParallelTestsGroup.builder,  i.e JLoadProfileUserGroups(JLoadProfileUsers userGroup, JLoadProfileUsers... userGroups)